docs/user-manual/wildcard-syntax.adoc

```diff
@@ -31,9 +31,9 @@ But `news.europe.#` would _not_ match:
 * `news.usa.sport`
 * `entertainment`
 
-== Matching a Single Word
+== Matching a Single or Partial Word
 
-The character `*` means "match a single word".
+The character `*` means "match a single or partial word".
```

Comment:
Its not clear whether the "single-word" configuration item (detailed lower down this documentation) which controls single word matches would also change this partial-word matching.